Example/Auth/Sample/MainViewController.m

```diff
@@ -1776,6 +1776,12 @@ - (void)signinWithProvider:(id<AuthProvider>)authProvider retrieveData:(BOOL)ret
           [self logFailure:@"sign-in with provider failed" error:error];
         } else {
           [self logSuccess:@"sign-in with provider succeeded."];
+          NSString *newUserString = authResult.additionalUserInfo.isNewUser ?
```

Comment:
Please check that addtionalUserInfo is non-nil before this like:

if (authResult.additionalUserInfo) {
  ...
  [self showMessagePromptWithTitle ...];
}